### PR TITLE
Feature/storage delete

### PR DESCRIPTION
**Pull request by aronerben**
Also adds indentation.

### CODE REVIEW

File: src/storage/storage_service.ml [hunk 1]
```diff
@@ -19,6 +19,13 @@ module Make (Log : Log.Sig.SERVICE) (Repo : REPO) : SERVICE = struct
     | None -> raise (Exception ("File not found with id " ^ id))
     | Some file -> Lwt.return file
 
+  let delete ctx ~id =
+    let* file = find ctx ~id in
+    let blob_id = StoredFile.blob file in
+    let* () = Repo.delete_blob ctx ~id:blob_id in
```
Comment (jerben) [inline src/storage/storage_service.ml:25]: @aronerben Please switch the calls and put them into `DbService.atomic`

**Review by jerben — CHANGES_REQUESTED** [review]

File: src/storage/storage_service.ml (hunk 1 above)
Comment (aronerben) [reply src/storage/storage_service.ml]: Done as you wish

**Comment by aronerben** [comment]
@jerben Ready for second review

File: src/storage/storage_service.ml [hunk 2]
```diff
@@ -22,9 +25,10 @@ module Make (Log : Log.Sig.SERVICE) (Repo : REPO) : SERVICE = struct
   let delete ctx ~id =
     let* file = find ctx ~id in
     let blob_id = StoredFile.blob file in
-    let* () = Repo.delete_blob ctx ~id:blob_id in
-    let* () = Repo.delete_file ctx ~id:file.file.id in
-    Lwt.return ()
+    DbService.atomic ctx (fun ctx ->
+        let* () = Repo.delete_file ctx ~id:file.file.id in
+        let* () = Repo.delete_blob ctx ~id:blob_id in
```
Comment (jerben) [inline src/storage/storage_service.ml:30]: @aronerben You could just return the value of the last call instead of doing `Lwt.return`

Comment (aronerben) [reply src/storage/storage_service.ml]: Cheers

**Pull request merged**